src/kraft_app.erl

```diff
@@ -1,10 +1,12 @@
-%%%-------------------------------------------------------------------
-%% @doc kraft public API
-%% @end
-%%%-------------------------------------------------------------------
-
 -module(kraft_app).
 
+-moduledoc """
+Kraft public API.
+
+This module is the entry point for the Kraft application. It starts the
+supervisor and stops the application.
```

Comment:
This is also not really a user facing module so I wouldn't document it at all at this point to be honest.